lib/msf/core/db_manager/import/burp_issue.rb

```diff
@@ -0,0 +1,34 @@
+require 'rex/parser/burp_issue_nokogiri'
+
+module Msf::DBManager::Import::BurpIssue
+  def import_burp_issue_noko_stream(args={},&block)
+    if block
+      doc = Rex::Parser::BurpIssueDocument.new(args,framework.db) {|type, data| yield type,data }
+    else
+      doc = Rex::Parser::BurpIssueDocument.new(args,self)
+    end
+    parser = ::Nokogiri::XML::SAX::Parser.new(doc)
+    parser.parse(args[:data])
+  end
+
+  def import_burp_issue_xml(args={}, &block)
+    bl = validate_ips(args[:blacklist]) ? args[:blacklist].split : []
+    wspace = args[:wspace] || workspace
+    if Rex::Parser.nokogiri_loaded
+      # Rex::Parser.reload("burp_issue_nokogiri.rb")
+      parser = "Nokogiri v#{::Nokogiri::VERSION}"
+      noko_args = args.dup
+      noko_args[:blacklist] = bl
+      noko_args[:wspace] = wspace
+      if block
+        yield(:parser, parser)
+        import_burp_issue_noko_stream(noko_args) {|type, data| yield type,data}
+      else
+        import_burp_issue_noko_stream(noko_args)
+      end
+      return true
+    else # Sorry
+      raise Msf::DBImportError.new("Could not import due to missing Nokogiri parser. Try 'gem install nokogiri'.")
```

Comment:
It looks like it is safe to remove it.

I looked into the code history a little bit. It looks like when Rex::Parser.nokogiri_loaded was introduced, we didn't ship Nokogiri, so at that time it made sense to check that. But since we do now, I would say we never need to use Rex::Parser.nokogiri_loaded ever again (unless we decide to stop shipping Nokogiri for some reason).